usr/src/uts/common/fs/zfs/zvol.c

```diff
@@ -1217,18 +1352,10 @@ zvol_dumpio(zvol_state_t *zv, void *addr, uint64_t offset, uint64_t size,
 int
 zvol_strategy(buf_t *bp)
 {
-	zfs_soft_state_t *zs = NULL;
```

Comment:
Again the style of the variable declarations seems to change for no reason.

Reply:
see my other comment (e.g. cleanup).